### PR TITLE
Created doc: WHERE

### PR DESCRIPTION
**Pull request by randomJoe211**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/8.clauses-and-options/where.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+        WHERE exists(v.age) \
+        RETURN v.name, v.age;
+
+```
+
+-->
+
+### Filter on edge rank
+
+In nGQL, if a group of edges has the same source vertex, destination vertex, and properties, the only thing that distinguishes them is the rank. Use rank conditions in `WHERE` to filter such edges.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/where.md:171]: if a group of edges has -> have

File: docs-2.0/3.ngql-guide/8.clauses-and-options/where.md [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+| "Tony Parker"    | 36    |
++------------------+-------+
+| "Tiago Splitter" | 34    |
++------------------+-------+
+Got 3 rows (time spent 4934/5832 us)
+```
+
+### Match any part of a string
+
+Use `CONTAINS "Pa"` to match a player name that ends with `Pa`.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/where.md:267]: ends with -> contains

**Review by Amber1990Zhang — COMMENTED** [review]
LGTM

File: docs-2.0/3.ngql-guide/8.clauses-and-options/where.md [hunk 3]
```diff
@@ -0,0 +1,386 @@
...
+
+## Syntax
+
+```ngql
+WHERE <expression> [ AND|OR|XOR <expression> ...])
+```
+
+## OpenCypher compatibility
+
+The following features are not supported in nGQL yet:
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/where.md:18]: The following features are not supported: TODO(planning)

File: docs-2.0/3.ngql-guide/8.clauses-and-options/where.md [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+
+* Filter on dynamically-calculated property, for example `WHERE v[toLower("AGE")] < 30`.
+* Use the `exists()` function to filter on the existence of a property.
+* Use patterns in `WHERE`.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/where.md:22]: Use graph patterns in `WHERE`. E.g  WHERE (v)--(v2)

**Review by whitewum — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**